### ISSUE TITLE
When I change the label text, the stamp no longer works

### ISSUE THREAD

**Issue by LouisePla** [pharo-graphics/Toplo#164]
When I change the label text, the stamp no longer works. 

**Example (code below)**
Before text update :
![image](https://github.com/pharo-graphics/Toplo/assets/123971303/faff7d7c-1f1d-447a-b491-0f65e788316d)
After text update the size isn't the size defined for the stamp :
![image](https://github.com/pharo-graphics/Toplo/assets/123971303/c8359b85-af9b-44d2-83eb-48c010516eec)

```smalltalk
|space panel label button|
label := ToLabel text: 'Hello'.
label addStamp: #H2.
button := ToButton new.
button label: (ToLabel text: 'Update').
button whenClickedDo: [ label text: 'Hello'. label textChanged  ].

panel := ToPane horizontal.
panel layout cellSpacing: 10.
panel addChild: button.
panel addChild: label.

space := BlSpace new.
space toTheme: ToBeeTheme new.
space root addChild: panel.
space show
```

**Comment by plantec** [comment]
normally fixed by commit 823b783

**Comment by LouisePla** [comment]
It’s perfect it works :) thank you!
